frontend/kesaseteli/employer/src/hooks/useApplicationsQuery.ts

```diff
@@ -0,0 +1,14 @@
+import { BackendEndpoint } from 'kesaseteli/employer/backend-api/backend-api';
+import handleResponse from 'kesaseteli/employer/backend-api/handle-response';
+import useBackendAPI from 'kesaseteli/employer/hooks/useBackendAPI';
+import Application from 'kesaseteli/employer/types/application';
+import { useQuery, UseQueryResult } from 'react-query';
+
+const useApplicationsQuery = (): UseQueryResult<Application[], Error> => {
+  const { axios } = useBackendAPI();
+  return useQuery<Application[], Error>('applications', () =>
+    handleResponse<Application[]>(axios.get(BackendEndpoint.APPLICATIONS))
```

Comment:
I noticed that `handleResponse` is always used in conjunction with `const { axios } = useBackendAPI();`.
I tucked it in the same hook to be used like:
`const { axios, handleResponse } = useBackendAPI();`

Reply:
great idea!